### PR TITLE
Update load syntax

### PR DESCRIPTION
**Pull request by aateeque**
Stab at what the changes in #1359 do

### CODE REVIEW

File: src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md [hunk 1]
```diff
@@ -21,15 +21,34 @@ Right now, only NuGet packages are supported.
...
 or
 #load "utilities.cake"
 ```
+Attempts to load `utilities.cake` from current directory.
+
+###### Local scheme:
+```csharp
+#l "local:?path=scripts\utilities.cake"
+or
+#load "local:?path=scripts\utilities.cake"
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md]: Should be a forward slash here `/`.

File: src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md [hunk 2]
```diff
@@ -21,15 +21,34 @@ Right now, only NuGet packages are supported.
...
 ```csharp
 #l "utilities.cake"
 or
 #load "utilities.cake"
 ```
+Attempts to load `utilities.cake` from current directory.
+
+###### Local scheme:
+```csharp
+#l "local:?path=scripts\utilities.cake"
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md]: Should be a forward slash here `/`.

File: src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md [hunk 3]
```diff
@@ -21,15 +21,34 @@ Right now, only NuGet packages are supported.
...
 
 #### Usage
-The directive has one parameter which is the location to the script.
+The directive has one parameter which is the location to the script which optionally includes a scheme: `local` or `nuget`. The default is `local`. 
 
+###### Default:
 ```csharp
 #l "utilities.cake"
 or
 #load "utilities.cake"
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md]: Perhaps should use `scripts/utilities.cake` like the `Local scheme` example below?

File: src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md [hunk 4]
```diff
@@ -21,15 +21,34 @@ Right now, only NuGet packages are supported.
...
 The load directive is used to reference external Cake scripts. Useful i.e. if you have common utility functions.
+Starting from 0.18.0 you can also load cake scripts from nuget.
 
 #### Usage
-The directive has one parameter which is the location to the script.
+The directive has one parameter which is the location to the script which optionally includes a scheme: `local` or `nuget`. The default is `local`. 
 
+###### Default:
 ```csharp
 #l "utilities.cake"
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs/fundamentals/preprocessor-directives.md]: Perhaps should use `scripts/utilities.cake` like the `Local scheme` example below?

**Comment by devlead** [comment]
@aateeque your changes have now been merged and will be available shortly.

**Pull request merged**